package.json

```diff
@@ -36,7 +36,7 @@
     "minimist": "^1.2.6",
     "package-dependency-graph": "^1.14.4",
     "prettier": "^3.0.0",
-    "typescript": "^5.2.2"
+    "typescript": "^5.4.5"
```

Comment:
TS update is needed for the latest `CSSImportRule` interface, as `supportsText` is fairly recent.